src/mainboard/sunxi/nezha/bt0/src/main.rs

```diff
@@ -583,12 +583,8 @@ extern "C" fn main() -> usize {
     }
 
     println!("Run payload at 0x{:x}", RAM_BASE);
-    unsafe {
-        let f: unsafe extern "C" fn() = transmute(RAM_BASE);
-        f();
-    }
-
-    // returns an address of dram payload; now cpu would jump to this address
+    
+	// returns an address of dram payload; now cpu would jump to this address
```

Comment:
Here's the proposal: Instead of this, remove the other part. Then it's less functions. WDYT?